tests/stubdata/output/jats_a+a_subtitle.json

```diff
@@ -660,6 +660,9 @@
RB host complex but to a nearby galaxy, and we suggested that a scenario of clumpy and merging galaxies could explain our observations.</p>\n<p>Further HST/F140W observations are planned at more than one year from the GRB trigger, so that any afterglow contribution can be firmly excluded and it will be possible to confirm the host properties such as the GRB SFR<sub>UV</sub> and the morphology. HST/F606W observations would allow us to identify which of the objects detected in the GRB proximity are at low redshift and therefore cannot be associated with the GRB host galaxy complex. Furthermore, VLT/MUSE observations would make it possible to confirm or discard the Lyman-<italic>\u03b1</italic> emission tentative detection, and, if confirmed, we would be able to analyze its position and extension, compared to the GRB afterglow position and the other objects in the field. These observations, combined with JWST data, to cover the host galaxy rest-frame optical emission, will offer the unique opportunity to combine the properties of the neutral and warm gas identified in this work with those of the continuum and ionized gas that would be determined through the galaxy photometry and the emission lines spectroscopy.</p>\n<p>This study shows the great potential of GRBs as tools to investigate the gas of very high-redshift galaxies. The metallicities obtained so far from the three GRB afterglows at <italic>z</italic>\u2004\u223c\u20046, for which these measurements have been possible, consistently show that high-<italic>z</italic> GRB hosts are already enriched to a metallicity of &gt; 2% solar, 1 Gyr after the Big Bang. Such metallicity enrichment is indeed expected if GRBs select those galaxies populating the faint-end of the galaxy luminosity function at these redshifts (<xref id=\"InR92\"/><xref ref-type=\"bibr\" rid=\"R92\">Salvaterra et al. 2012</xref>, <xref ref-type=\"bibr\" rid=\"R93\">2013</xref>), which are believed to provide the major contribution to cosmic reionization. It would be very difficult, if not impossible, to obtain the information presented in this work for field galaxies observed even with the best telescopes of the future (JWST, ELT,...). To increase the number of these kind of studies, it is fundamental to develop space satellites with energy range coverage and sensitivity suitable to detect the very high-redshift GRB population (<xref id=\"InR42\"/><xref ref-type=\"bibr\" rid=\"R42\">Ghirlanda et al. 2015</xref>; e.g., THESEUS, Gamow; <xref id=\"InR2\"/><xref ref-type=\"bibr\" rid=\"R2\">Amati et al. 2018</xref>; <xref id=\"InR107\"/><xref ref-type=\"bibr\" rid=\"R107\">Tanvir et al. 2021a</xref>; <xref id=\"InR126\"/><xref ref-type=\"bibr\" rid=\"R126\">White et al. 2021</xref>, respectively). Onboard NIR afterglow detection and preselection of very high-redshift GRBs would be a significant added value to optimize ground-based spectroscopic observations with sensitive medium-high resolution spectrographs.</p>\n</sec>\n</body>"
```

Comment:
Are all of these test files safe (non-copyrighted) to have up in a public repo on Github?